libquickevent/libquickeventcore/src/utils.cpp

```diff
@@ -5,26 +5,10 @@
 namespace quickevent {
 namespace core {
 
- QString Utils::dateTimeToIsoStringWithUtcOffset(const QDateTime &dt)
+ QString Utils::dateTimeToIsoStringWithUtcOffset(QDateTime dt)
 {
-	QString ret = dt.toString(Qt::ISODate);
-	if(dt.timeSpec() == Qt::LocalTime) {
-		int offset_min = dt.offsetFromUtc();
-		if(offset_min == 0) {
-			ret += 'Z';
-		}
-		else {
-			if(offset_min < 0) {
-				ret += '-';
-				offset_min = -offset_min;
-			}
-			else {
-				ret += '+';
-			}
-			ret += QStringLiteral("%1:%2").arg(offset_min / (60 * 60), 2, 10, QChar('0')).arg((offset_min / 60) % 60, 2, 10, QChar('0'));
-		}
-	}
-	return ret;
+	dt.setOffsetFromUtc(dt.offsetFromUtc());
```

Comment:
neni toto idempotence?

Reply:
Its not a bug, its a feature. :)

https://bugreports.qt.io/browse/QTBUG-26161?focusedCommentId=554227&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-554227

Tested it does not work - the time zone offset is missing - without this line. (at least as for today).

Reply:
interesting :)

Reply:
please add comment with short explanation of this strange line.